client/components/payment-activity/index.tsx

```diff
@@ -65,10 +66,12 @@ const PaymentActivity: React.FC = () => {
 	const isOverviewSurveySubmitted =
 		wcpaySettings.isOverviewSurveySubmitted ?? false;
 
+	const { selectedCurrency } = useSelectedCurrency();
+
 	const { paymentActivityData, isLoading } = usePaymentActivityData( {
 		// In future this will be bound to currency picker via useSelectedCurrency().
```

Comment:
We may want to remove this comment? Since we are already attaching this to useSelectedCurrency.

Reply:
Sorry, I meant the comment on the line 72 and 73 

```
// In future this will be bound to currency picker via useSelectedCurrency()....
```